File: exercicios/2006/2006.py
def cha():
    vchar = int(input())
    return(vchar)

def competidor(vchar):
  
    qt = 0
  
    v = input()
    v = v.split(' ')

    for i in range(5):
        if int(v[i]) == vchar:
            qt += 1
        i += 1
    return(qt)

def main():
    qt = competidor(cha())

    print(qt)

main()

File: exercicios/JavaScript/2006/2006.js
var input = require("fs").readFileSync("stdin","utf-8");
var lines = input.split("\n");

var tipo_cha = parseInt(lines.shift());
var total = 0;
var voto = lines.shift().split(" ");

voto.forEach(voto => {
    if(voto == tipo_cha){
        total ++
    }
});

console.log(total)